packages/victory-bar/src/bar.js

```diff
@@ -107,17 +108,18 @@ const Bar = (props) => {
     ? getPolarBarPath(props, cornerRadius)
     : getBarPath(props, barWidth, cornerRadius);
   const defaultTransform = polar && origin ? `translate(${origin.x}, ${origin.y})` : undefined;
-
   return React.cloneElement(props.pathComponent, {
     ...props.events,
     style,
     d: path,
-    transform: props.transform || defaultTransform,
+    "aria-label": props.ariaLabel,
```

Comment:
this applies to the html element as ariaLabel if not defined specifically as "aria-label" here, which I think it needs to be read properly by screen readers... lmk if you know otherwise

Reply:
Yep, I think it's correct to have `aria-*` props dash cased in JSX